js/repeater-list.js

```diff
@@ -687,8 +689,14 @@
 		var chevUp = 'glyphicon-chevron-up';
 		var $div = $('<div class="repeater-list-heading"><span class="glyphicon rlc"></span></div>');
 		var checkAllID = (this.$element.attr('id')+'_' || '') + 'checkall';
-		var checkBoxMarkup = '<div class="repeater-list-heading header-checkbox"><div class="checkbox checkbox-inline"><input type="checkbox" id="' + checkAllID + '">'+
-			'<label for="' + checkAllID + '"></label></div></div>';
+
+		var checkBoxMarkup = '<div class="repeater-list-heading header-checkbox">' +
+				'<label for="' + checkAllID + '" class="checkbox-custom checkbox-inline">' + 
+					'<input id="' + checkAllID + '" class="sr-only" type="checkbox" value="">' +
```

Comment:
Sounds good. I knew that but was trying to make I didn't break things. Will update and test.